src/vmath.nim

```diff
@@ -79,6 +79,12 @@ func `-`*(a: Vec2): Vec2 =
   result.x = -a.x
   result.y = -a.y
 
+func hash*(a: Vec2): Hash =
+  var h: Hash
+  h = h !& hash(a.x)
+  h = h !& hash(a.y)
+  result = !$h
```

Comment:
can you do "no return to be consistent with rest of code in vmath"

Reply:
done, caught some others too